### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by hl2guide** [ncosgray/cuppa_mobile#30]
Please consider adding a dark mode.

**Comment by ncosgray** [comment]
Cuppa supports dark mode on both Android and iOS. The app follows your system settings; when dark mode is enabled at the OS level, Cuppa will follow suit and use its dark mode.

Can you clarify the issue? Are you asking to add a preference that forces Cuppa to always use dark more?

**Comment by hl2guide** [comment]
Yes, please add a preference that forces Cuppa to always use dark mode.

**Comment by mondlicht-und-sterne** [comment]
> Yes, please add a preference that forces Cuppa to always use dark mode.

That would be great.

**Comment by aha999** [comment]
in my opinion, it would be best if Cuppa would, by default follow a system theme setting, and also have manual light/dark force settings. This way, it would automatically be set per user preference of a general system theme.

**Comment by hl2guide** [comment]
Especially for older devices with no OS level dark mode setting.
